gobblin-api/src/main/java/org/apache/gobblin/util/ClassAliasResolver.java

```diff
@@ -100,15 +101,17 @@ public String resolve(final String possibleAlias) {
    * and is also a subclass of {@link #subtypeOf}, if it fails it returns a class object for name
    * <code>aliasOrClassName</code>.
    */
-  public Class<? extends T> resolveClass(final String aliasOrClassName) throws ClassNotFoundException {
+  public Class<? extends T> resolveClass(final String aliasOrClassName)
+      throws ClassNotFoundException {
```

Comment:
Is this something controlled by the gobblin codestyle file or by the intellij settings?

Reply:
I think it's part of Gobblin code style settings, and IntelliJ automatically reformats the code to this.

Reply:
Should we keep this style?